docs/en/02_Developer_Guides/09_Security/04_Secure_Coding.md

```diff
@@ -585,6 +583,88 @@ In a future release this behaviour will be changed to be on by default, and this
+flag is not present, the browser will send the cookie even if HTTPS is not in use, which means it is transmitted in 
+clear text and can be intercepted and stolen by an attacker who is listening on the network.
+
+- The `HTTPOnly` flag lets the browser know whether or not a cookie should be accessible by client-side JavaScript 
+code. It is best practice to set this flag unless the application is known to use JavaScript to access these cookies 
+as this prevents an attacker who achieves cross-site scripting from accessing these cookies.
+
+
+	:::php
+	
+	Cookie::set('cookie-name', 'chocolate-chip', $expiry = 30, $path = null, $domain = null, $secure = true, 
+		$httpOnly = false
+	);
+
+## Security Headers
+
+In addition to forcing HTTPS browsers can support additional security headers which can only allow access to a website 
+via a secure connection. As browsers increasingly provide negative feedback regarding unencrypted HTTP connections, 
+ensuring an HTTPS connection will provide a better and more secure user experience.  
+
+- The `Strict-Transport-Security` header instructs the browser to record that the website and assets on that website 
+MUST use a secure connection. This prevents websites from becoming insecure in the future from stray absolute links 
+or references without https from external sites. Check if your browser supports [HSTS](https://hsts.badssl.com/)
+  - `max-age` can be configured to anything in seconds: `max-age=31536000` (1 year), for roll out, consider something 
+  lower
+  - `includeSubDomains` to ensure all present and future sub domains will also be HTTPS
+
+For sensitive pages, such as members areas, or places where sensitive information is present, adding cache control
+ headers can explicitly instruct browsers not to keep a local cached copy of content and can prevent content from 
+ being cached throughout the infrastructure (e.g. Proxy, caching layers, WAF etc). 
+ 
+- The headers `Cache-control: no-store` and `Pragma: no-cache` along with expiry headers of `Expires: <current date>` 
+and `Date: <current date>` will ensure that sensitive content is not stored locally or able to be retrieved by 
+unauthorised local persons. SilverStripe adds the current date for every request, and we can add the other cache 
+ headers to the request for our secure controllers:
+
+
+	:::php
+
+	class MySecureController extends Controller {
+		
+		public function init() {
+			parent::init();
+        	
+			// Add cache headers to ensure sensitive content isn't cached.
+			$this->response->addHeader('Cache-Control', 'max-age=0, must-revalidate, no-transform');
+			$this->response->addHeader('Pragma', 'no-cache'); // for HTTP 1.0 support
+
+			HTTP::set_cache_age(0);
+			HTTP::add_cache_headers($this->response);
+			
+			// Add HSTS header to force TLS for document content
+			$this->response->addHeader('Strict-Transport-Security', 'max-age=86400; includeSubDomains');
```

Comment:
I'd note that the use of `includeSubDomains` is quite risky and  I've had this catch me out before. Lots of devs will work on projects where they don't have control or the ability to enable https on all of an organisations sub-domains so this can be risky.